src/Bot.ts

```diff
@@ -92,14 +93,44 @@ export class Bot extends builder.UniversalBot {
     }
 
     // set incoming event to any because membersAdded is not a field in builder.IEvent
-    private getConversationUpdateHandler(bot: builder.UniversalBot): (event: any) => void {
-        return async function(event: any): Promise<void> {
+    private getConversationUpdateHandler(bot: builder.UniversalBot): (event: builder.IConversationUpdate) => void {
+        return async function(event: builder.IConversationUpdate): Promise<void> {
+            // We are only interested in member add events
+            if (!event.membersAdded || (event.membersAdded.length === 0)) {
+                return;
+            }
+
             let session = await loadSessionAsync(bot, event);
 
-            if (event.membersAdded && event.membersAdded[0].id && event.membersAdded[0].id.endsWith(config.get("bot.botId"))) {
-                session.send(Strings.bot_introduction); // probably only works in Teams
+            // Determine if the bot was added to the conversation
+            let botId = event.address.bot.id;
+            let botAdded = event.membersAdded && event.membersAdded.find(member => (member.id === botId));
```

Comment:
Little pedantic, but can this be named like the C# example and be botWasAdded so it can't be confused with something else like "the bot's id that was added"?